src/Views/blades/pb-footer.blade.php

```diff
@@ -3,13 +3,10 @@
 		<div class="pull-left">
 			<ul class="list-inline" style="padding-top:1em;">
 				<li>
-					<a href="{{env('APP_URL')}}/feedback">Feedback</a>
-				</li>
-				<li>
 					<a href="{{env('APP_URL')}}/release">Release Information</a>
 				</li>
 				<li>
-					<a href="{{env('APP_URL')}}/help">Help</a>
+					<a href="{{env('APP_URL')}}/docs">Documentation</a>
 				</li>
```

Comment:
In this case, I think 'Help' just means the documentation for using the app. Since the widget has the text, 'Help' I wanted to change this.